docs/02-guides/01-static-website/index.md

```diff
@@ -0,0 +1,67 @@
+---
+sidebar_label: Static Website
+title: Deploying an Static Website with Easypanel
+description: Deploying an Static Website with Easypanel guide
+---
+
+# Hosting a Static Website with Easypanel
+
+In this tutorial, we will be guiding you through the process of hosting a static website on Easypanel. You have two options available: using a Git repository or a Docker image. Let's dive into the steps involved:
+
+## Prerequisites
+
+Before we begin, make sure you have the following:
+
+- An existing static website project.
+- Easypanel account credentials.
+
+## Step 1 - Creating a New Project
+
+Firstly, log in to your Easypanel account and create a new project:
+
+- Click on the "New" button and select the "App" option.
+
+![Environment Setup](./new-project.png)
+
+## Step 2 - Configuring General Settings
```

Comment:
in other quickstarts we have "Configuring the Git/GitHub Source". Here you can use "Configuring the Source"